crates/op-rbuilder/src/payload_builder.rs

```diff
@@ -93,6 +94,8 @@ pub struct CustomOpPayloadBuilder {
     flashblocks_ws_url: String,
     chain_block_time: u64,
     flashblock_block_time: u64,
+    #[allow(dead_code)]
```

Comment:
```suggestion
    #[expect(dead_code)]
```
prefer using `expect` so compiler helps you to clean up redundant attributes